loopy/kernel/instruction.py

```diff
@@ -409,6 +411,50 @@ def __setstate__(self, val):
         self.within_inames = (
                 intern_frozenset_of_ids(self.within_inames))
 
+    def get_domain(self, kernel):
```

Comment:
I'd prefer if this were a function (rather than a method).

Reply:
Agreed, making it a method wasn't a great choice. dc57c07d